controllers/activemqartemis_controller.go

```diff
@@ -223,6 +227,19 @@ func validateExtraMounts(customResource *brokerv1beta1.ActiveMQArtemis, client r
 			}, nil
 		}
 	}
+	loggingConfigMapName := customResource.Name + "-logging-config"
```

Comment:
If we use "-logging-config" as a special postfix, we can just look for entries matching that in the extraMounts and use that as the trigger to populate the env var. We don't need the crd.name prefix any more, just something ending with "-logging-config".
In this way, the reference is explicit in the CR and we don't need to watch config maps any more. It will be simpler.

Reply:
yeah agreed